fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -2615,6 +2625,7 @@ type
     | arrayType
     | structType
     | mapType
+    | aggStateType
```

Comment:
I think we need to discuss the type first.
I don't think there is a need to introduce a new type, we can use binary here.

Reply:
This pr doesn't add a new type instead.

You can see it deeply, this is only a syntax sugar which wraps an aggregate function's intermediate type which is not seen by users.